api/src/config.rs

```diff
@@ -31,6 +31,12 @@ pub struct BootloaderConfig {
     /// Configuration for changing the level of the filter of the messages that are shown in the
     /// screen when booting. The default is 'Trace'.
     pub log_level: LevelFilter,
+
+    /// Configuration for enable or disable the framebuffer logger.
+    pub frame_buffer_logger_status: LoggerStatus,
+
+    /// Configuration for enable or disable the serial being logger.
+    pub serial_logger_status: LoggerStatus,
```

Comment:
Minor wording suggestions:

```suggestion

    /// Whether the bootloader should print log messages to the framebuffer when booting.
    ///
    /// Enabled by default.
    pub frame_buffer_logger_status: LoggerStatus,

    /// Whether the bootloader should print log messages to the serial port when booting.
    ///
    /// Enabled by default.
    pub serial_logger_status: LoggerStatus,
```